eventlivedata/live_db.go

```diff
@@ -0,0 +1,65 @@
+package eventlivedata
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"os"
+	"strconv"
+)
+
+type live_db_adapter interface {
+	initConn() error
+	getLiveHeatMap(event int) (map[string]int, error)
+}
+
+type dynamodbAdaptor struct {
+	db         *dynamodb.DynamoDB
+	streamName string
+}
+
+// initConn opens the connection to the location event kinesis queue
+func (db *dynamodbAdaptor) initConn() error {
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String("eu-central-1")},
+	)
+	if err != nil {
+		fmt.Println("Got error creating session:")
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	// Create DynamoDB client
+	db.db = dynamodb.New(sess)
+
+	return nil
+}
+
+// Pre: the event object is valid
+func (db *dynamodbAdaptor) getLiveHeatMap(event int) (map[string]int, error) {
+	params := &dynamodb.ScanInput{
+		TableName: aws.String("current_position"),
```

Comment:
Could we store table names as constants?

Reply:
fixed in e289132